modulemd/include/private/glib-extensions.h

```diff
@@ -14,10 +14,46 @@
 #pragma once
 
 #include <glib.h>
+#include <glib/gtypes.h>
 
 #include "config.h"
 
 /* GDate autoptr cleanup was finally added in GLib 2.63.3. */
 #ifndef HAVE_GDATE_AUTOPTR
 G_DEFINE_AUTOPTR_CLEANUP_FUNC (GDate, g_date_free)
 #endif
+
+#ifndef HAVE_EXTEND_AND_STEAL
+
+#ifndef MIN_ARRAY_SIZE
```

Comment:
Arg. Sorry, I just noticed `MIN_ARRAY_SIZE` was also privately within the upstream `glib/garray.c` file, not in a header. It should really go into `modulemd/modulemd-util.c` as well.